src/activate/activate.c

```diff
@@ -385,6 +396,11 @@ static int parse_argv(int argc, char *argv[]) {
                 return -EINVAL;
         }
 
+        if (arg_datagram && arg_accept) {
+                log_error("Datagram sockets does not accept connections. Use -a option only with stream sockets.");
```

Comment:
Grammar nit: `Datagram sockets do not accept connections. The -a option can only be used with stream sockets.`